### DISCUSSION TITLE
Better sorting labels, custom columns

### DISCUSSION THREAD

**Discussion by poeterdebier** [jonaswinkler/paperless-ng#1196] [Feature Requests]
Hi,

I like paperless-ng alot. A problem however is that using a lot of labels for a document will eventually make it difficult to select correct labels.

How would you handle a document like this: 

"Micrologix 1400 reference manual" I would like to label this with the following labels: micrologix, manual, allen-bradley, automation systems and even a couple more when this belongs to a certain unique system/device.

One solution that would really work for me is to be able to add separate columns like correspondent and document type. I can then just make colums for maker, type, system etc etc. 

So actually I am asking for the function to add your own colums.

regards Piet

**Comment by TheFehr** [comment]
Probably duplicate of #274
